integration_tests/test_gridworkflow.py

```diff
@@ -0,0 +1,10 @@
+from datacube.api.grid_workflow import GridWorkflow
+
+
+def test_create_gridworkflow_with_logging(index):
+    from logging import getLogger, StreamHandler
+
+    logger = getLogger(__name__)
+    logger.addHandler(StreamHandler())
```

Comment:
Since this changes global scope, we should add a finally block that removes it again after doing the test 

(alternatively it could be done with a pytest fixture but that's probably overkill in this case)